integration_tests/test_rough_align_qc.py

```diff
@@ -0,0 +1,97 @@
+import os
+import pytest
+import logging
+import renderapi
+import json
+import glob
+from six.moves import urllib
+from test_data import (MONTAGE_SCAPES_TSPECS,
+                       ROUGH_QC_TEST_PT_MATCHES,
+                       ROUGH_QC_OUT_STACK,
+                       render_params,
+                       solver_montage_parameters as solver_example,
+                       apply_rough_alignment_example as ex1,
+                       pool_size)
+
+from rendermodules.module.render_module import RenderModuleException
+from rendermodules.dataimport.make_montage_scapes_stack import MakeMontageScapeSectionStack, create_montage_scape_tile_specs
+from rendermodules.em_montage_qc.rough_align_qc import RoughAlignmentQC
+from rendermodules.solver.solve import Solve_stack
+
+
+@pytest.fixture(scope='module')
+def render():
+    render_params['project'] = 'rough_align_test'
+    render = renderapi.connect(**render_params)
+    return render
+
+@pytest.fixture(scope='module')
+def downsample_stack(render):
+    pre_rough_stack = "pre_rough_stack"
+    
+    renderapi.stack.create_stack(pre_rough_stack, render=render)
+    renderapi.client.import_tilespecs(pre_rough_stack, MONTAGE_SCAPES_TSPECS, render=render)
+    renderapi.stack.set_stack_state(pre_rough_stack, "COMPLETE", render=render)
+    yield pre_rough_stack
+    #render.run(renderapi.stack.delete_stack, pre_rough_stack)
```

Comment:
These stacks should be cleaned up after the scope ends.

Reply:
uncommented the line